### PR TITLE
Update navbar, new about, contact, and products pages.

### PR DESCRIPTION
**Pull request by schlosser**

### CODE REVIEW

File: src/screens/Government/Government.tsx [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+                <a href="mailto:gov@covidactnow.org">gov@covidactnow.org</a>.
+              </Typography>
+            </li>
+            <li>
+              <Typography variant="body1" component="p">
+                <strong>Use our API</strong> <br />
+                You can find our API{' '}
+                <a href="https://github.com/covid-projections/covid-data-model/blob/master/api/README.V1.md">
+                  here
+                </a>
```
Comment (asciimike) [inline src/screens/Government/Government.tsx:52]: For some reason the link isn't showing up properly on the preview:

image

Comment (schlosser) [reply src/screens/Government/Government.tsx:52]: Updated! Link was old.

**Comment by asciimike** [comment]
LGTM

**Comment by asciimike** [comment]
@igorkofman or @zjrosen1 would you mind merging?

**Comment by igorkofman** [comment]
Looks great! 
Functionality:
- "The Model" link doesn't work.
- "Check out our press packet here, or reach out to us directly." doesn't link to a press packet

Eng: 
- Do we really need the fancy scrolling implementation? It's a bit glitchy. Can with just use {smooth:true}?

Design:
- The Georgetown image feels awkwardly placed and too small - could it be more clearly associated with the "we work in association with"? 
- The page content feels kind of narrow compared to the rest of the site - but it's probably just me
- I'd have preferred plain email address links to buttons as looking more professional, but maybe that's just me as well

PS. Did we fail to merge this a week ago?   Pls ping if you're waiting for a merge and it doesn't happen!

**Comment by schlosser** [comment]
Functionality + Eng feedback addressed. I've widened the gutters and removed buttons as well. Leaving the Georgetown logo, as we've had a couple conversations about it.

**Pull request merged**